### ISSUE TITLE
Game crashes when dye from tinted is used

### ISSUE THREAD

**Issue by Aero48** [MehVahdJukaar/labels-multi#10]
The game freezes and crashes if any dye from the tinted mod is used on a label.
Log: [crash-2022-11-16_17.46.02-client.txt](https://github.com/MehVahdJukaar/labels-multi/files/10026250/crash-2022-11-16_17.46.02-client.txt)

**Comment by Tander** [comment]
Yep, applying dye from Tinted crashes client dye rendering error, which makes player unable to log in into world anymore
[crash-2022-12-14_07.38.48-client.txt](https://github.com/MehVahdJukaar/labels-multi/files/10224400/crash-2022-12-14_07.38.48-client.txt)

**Comment by MehVahdJukaar** [comment]
Should be fixed. Tinted stuff will use the dye text color
